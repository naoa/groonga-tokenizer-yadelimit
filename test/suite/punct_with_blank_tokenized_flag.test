register tokenizers/yadelimit

tokenize TokenYaDelimitPunctWithBlank "a.B￾cDe 1￾23" NormalizerAuto ENABLE_TOKENIZED_DELIMITER
