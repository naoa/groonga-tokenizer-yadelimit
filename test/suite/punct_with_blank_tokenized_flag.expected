register tokenizers/yadelimit
[[0,0.0,0.0],true]
tokenize TokenYaDelimitPunctWithBlank "a.B￾cDe 1￾23" NormalizerAuto ENABLE_TOKENIZED_DELIMITER
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "a.b",
      "position": 0
    },
    {
      "value": "cde 1",
      "position": 1
    },
    {
      "value": "23",
      "position": 2
    }
  ]
]
